ruby_review_week_four.rb

```diff
@@ -1,37 +1,135 @@
 # You may NOT use Google. These questions are just to test where you are at. Get through as many of these as you can. If you don’t get through all of them it’s ok - they are designed to take longer than the available time. If you can’t remember a bit of code, write the pseudo-code that represents what you would write your Ruby code.
 
 # 1. What is a variable?
+can store data.
```

Comment:
store data is correct, when you define a variable also mention that you can re-use the variable by referencing its name